### DISCUSSION TITLE
How to use custom icon

### DISCUSSION THREAD

**Discussion by sohailahmad07** [filamentphp/filament#14988] [Help]
### Package

Table builder

### Package Version

3.2.124

### How can we help you?

Hi i have this action on a table and i am passing the icon and HTML string but i seem to render it as image
any idea on how i can using my font awesome pro icons
```php
  ActionGroup::make([
  EditAction::make()
                    ->icon(new HtmlString(Blade::render('<i class="fal fa-edit"></i>')))
                    ->fillForm(fn(Organization $organization) => $organization->toArray())
                    ->form([
                        TextInput::make('name'),
                        TextInput::make('address'),
                        Textarea::make('description'),
                    ])
 ])
```

![Screenshot 2024-12-03 at 19-05-14 Incrediverse](https://github.com/user-attachments/assets/18806308-8bdb-4690-b690-9bf6d6c56721)

**Comment by leandrocfe** [comment]
https://github.com/owenvoke/blade-fontawesome

```php
->icon('far-pen-to-square')
```
